unofficial_components/playbooks/odh-test/kserve-e2e-test-with-existing-cluster/config.yaml

```
@@ -0,0 +1,120 @@
+playbook:
+  name: kserve-execute-e2e-test-with-existing-cluster
+  created_date: "20250620"
+  description: |
+    This playbook configure test cluster that install all required objects for e2e tests and run test.
+
+    By default, it will use the fast channel of kserve/odh-model-controller. If you want to test with custom build, you can set the following cases:
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)